File: Python/Program/Store.py
from Business import Business

class Store(Business):
    def __init__(self, owner: str, name: str, address: str, mainly_sell: str, opening_day: list, opening_hour: tuple):
        super().__init__(owner, name, address)
        self._mainly_sell = mainly_sell
        self._opening_day = opening_day
        self._opening_hour = opening_hour

    def get_mainly_sell(self) -> str:
        return self._mainly_sell

    def set_mainly_sell(self, mainly_sell: str):
        self._mainly_sell = mainly_sell

    def get_opening_day(self) -> list:
        return self._opening_day

    def set_opening_day(self, opening_day: list):
        self._opening_day = opening_day

    def get_opening_hour(self) -> tuple:
        return self._opening_hour

    def set_opening_hour(self, opening_hour: tuple):
        self._opening_hour = opening_hour

File: C++/Program/Store.cpp
#pragma once
#include <iostream>
#include <string>
#include <vector>
#include "Business.cpp"

using namespace std;

class Store : public Business {
private:
    string mainlySell;
    vector<string> openingDay;
    pair<int, int> openingHour;

public:
    // Constructor
    Store(){}

    Store(string owner, string name, string address, string mainlySell, vector<string> openingDay, pair<int, int> openingHour)
        : Business(owner, name, address) {
        this->mainlySell = mainlySell;
        this->openingDay = openingDay;
        this->openingHour = openingHour;
    }

    // Getter and Setter for mainlySell
    string getMainlySell() {
        return mainlySell;
    }

    void setMainlySell(string mainlySell) {
        this->mainlySell = mainlySell;
    }

    // Getter and Setter for openingDay
    vector<string> getOpeningDay() {
        return openingDay;
    }

    void setOpeningDay(vector<string> openingDay) {
        this->openingDay = openingDay;
    }

    // Getter and Setter for openingHour
    pair<int, int> getOpeningHour() {
        return openingHour;
    }

    void setOpeningHour(pair<int, int> openingHour) {
        this->openingHour = openingHour;
    }
};